src/main/java/teammates/ui/template/StudentFeedbackResultsQuestionWithResponses.java

```diff
@@ -21,4 +22,28 @@ public FeedbackResultsQuestionDetails getQuestionDetails() {
         return responseTables;
     }
 
+    public List<FeedbackResultsResponseTable> getSelfResponseTables() {
```

Comment:
You can do some fancy stuff. :P. It's Java8 now.
```java
selfResponseTables = responseTables.stream().filter(responseTable -> responseTable.isGiverNameYou()).collect(Collectors.toList());
```

Also please add Javadoc for the method. A simple method summary is enough.